content/2024-01-17_jak-jsme-z-tezitek-udelali-uzitecne-notebooky-pro-zaky.md

```diff
@@ -0,0 +1,41 @@
+Title: Jak jsme z těžítek udělali užitečné notebooky pro žáky
+Date: 2024-01-17 13:55:00
+Modified: 2024-01-17 13:55:00
+Author: Lumír Balhar
+Twitter: lumirbalhar
+Gravatar: frenzy.madness@gmail.com
+About: Lumír pracuje jako senior software inženýr a ve volném čase organizuje komunitní srazy, kurzy a workshopy, vyučuje první pomoc a dělá dobrovolného hasiče.
+
+V moderní době výkonného a kvalitního hardware není výjimkou, když telefon, tablet nebo notebook, který je jinak stále ve skvělé kondici, dojde na konec svého životního cyklu, protože přestane dostávat softwarové aktualizace. To se nejprve projeví na horším zabezpečení a následně i na použitelnosti, protože moderní aplikace přestávají na neaktualizovaném systému jedna po druhé fungovat. Funkční zařízení se ale přece nevyhazují a tak by se na školách po celé republice jistě našla nejedna skříň plná zařízení, která sice fungují, ale nikdo je nepoužívá.
+
+![clipart starý počítač]({static}/images/old_pc_clipart.png)
+
+Nejiná byla situace i na GALILEO SCHOOL - bilingvní mateřské školé a základní školé, s.r.o. ve Frýdku-Místku, kde měli k dispozici téměř čtyři desítky skoro deset let starých chromebooků Thinkpad. Jejich baterie byla stále ve skvělé kondici, procesor a další komponenty na kancelářskou práci dostačující, ale v zastaralém prohlížeči už moderní kancelářské balíky odmítly fungovat. Místní učitelku a absolventku začátečnického kurzu programování v komunitě PyLadies Ostrava Zuzku Grosmanovou napadlo, že tento problém musí mít nějaké řešení a tak se ozvala ostravské komunitě a započalo hledání řešení.
+
+V tuto chvíli by se mohlo zdát, že reinstalaci operačního systému nebude nic bránit a nebude tedy ani o čem referovat. Opak je pravdou. Chromebooky mají firmware na základní desce chráněn proti přepisu a tedy jej nejde jednoduše nahradit. Bez nového fimware ale nejde provést reinstalaci operačního systému, jak jsme zvyklí z obyčejných počítačů.
```

Comment:
Čárka před a nebude tedy. Dvakrát. (Doporučuji druhé a tedy nahradit jinou spojkou.)

Reply:
Druhé tedy nahrazeno v posledním fixup commitu. Nerozumím důvodům pro čárky. 

```
V tuto chvíli by se mohlo zdát, že reinstalaci operačního systému nebude nic bránit a nebude tedy ani o čem referovat. Opak je pravdou. Chromebooky mají firmware na základní desce chráněn proti přepisu a nelze jej jednoduše nahradit.
```

V první větě mi `že reinstalaci…` nepřipadá jako věta vložená. Jaký je důvod dát před `a nebude tedy…` čárku?

V druhé větě tomu rozumím ještě méně. Je to jednoduché souvětí se spojkou a.

Reply:
> ### Poměr důsledkový (*a proto, a tedy, a tak, a tudíž, a tím* apod.)
> 
> Při vyjadřování důsledkového poměru spojka *a* zpravidla stojí ve spojeních *a proto, a tedy, a tak, a tudíž*: *Nedostali jsme zprávu včas, a proto jsme nemohli přijít. Ti lidé neplatí nájemné již několik let, a tak mají být vystěhováni. Dostatek tepla lze zajistit i poměrně malým, a tedy levným tepelným čerpadlem. Žádná překážka mu nestála v cestě, a tudíž nepociťoval strach.*
> 
> ...
> 
> Podobně se spojovací funkce spojení *a + proto, tak, tedy, tudíž* oslabuje, stojí‑li výrazy *proto, tak, tedy* a *tudíž* uvnitř věty. Psaní čárky v tomto případě není zcela závazné, hovoříme o slabém důsledkovém poměru a můžeme volit psaní s čárkou i bez ní: *Investoval uváženě(,) a přišel si tak na slušný zisk. Jednání s nájemcem jsem hned napoprvé pokazil(,) a rozhodl jsem se tedy, že to zkusím jinde. Máte diplomatické pasy(,) a můžete tudíž žádat zrušení prohlídky zavazadel.*

https://prirucka.ujc.cas.cz/?id=153#nadpis5

První *a nebude tedy* má podle všeho čárku volitelnou (mea culpa, ale opravdu bych jí tam dal).
Druhé *a tedy* mělo čárku povinnou, ale už tam není, teď je tam poměr slučovací bez čárky. (Stylisticky by podle mě bylo lepší nechat důsledkový, např. *a proto* místo původního opakujícího se *a tedy*).

Reply:
Děkuji, fixnuto.